datafusion/sqllogictest/test_files/joins.slt

```diff
@@ -1778,7 +1778,7 @@ AS VALUES
 ('BB', 6, 1);
 
 query TII
-select col1, col2, coalesce(sum_col3, 0) as sum_col3
+select col1, col2, coalesce(sum_col3, arrow_cast(0, 'UInt64')) as sum_col3
```

Comment:
maybe also we can change this test back too

Reply:
I think this will fail 🤔 since I disable the coercion between i64 and u64, but I'm not 100% sure